source/installation-guide/packages-list.rst

```diff
@@ -321,3 +322,67 @@ MSU
++      DEB      +                                    +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|               |              Ubuntu                |              | `wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_AARCH64|_arm64.deb <|DEB_AGENT_DEBUG_SYMBOLS|_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_AARCH64|_arm64.deb>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_AARCH64|_arm64.deb.sha512>`__)                   |
++               +                                    +    aarch64   +--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|               |                                    |              | `wazuh-manager-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_MANAGER_AARCH64|_arm64.deb <|DEB_MANAGER_DEBUG_SYMBOLS|_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_MANAGER_AARCH64|_arm64.deb>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-manager-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_MANAGER_AARCH64|_arm64.deb.sha512>`__)           |
++               +                                    +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|               |            Raspbian OS             |     armhf    | `wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_ARMHF|_armhf.deb <|DEB_AGENT_DEBUG_SYMBOLS|_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_ARMHF|_armhf.deb>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_ARMHF|_armhf.deb.sha512>`__)                   |
++               +                                    +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|               |                                    |   Power PC   | `wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb <|DEB_AGENT_DEBUG_SYMBOLS|_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb.sha512>`__)             |
```

Comment:
Change arch to match https://github.com/wazuh/wazuh-documentation/pull/5090 approach
```suggestion
|               |                                    |   powerpc   | `wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb <|DEB_AGENT_DEBUG_SYMBOLS|_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-dbg_|WAZUH_LATEST|-|WAZUH_REVISION_DEB_AGENT_PPC64|_ppc64el.deb.sha512>`__)             |
```